src/aind_metadata_service/slims/models.py

```diff
@@ -253,40 +295,77 @@ class ContentsTableRow(BaseModel):
         None, title="Original content barcode"
     )
 
-    @staticmethod
-    def map_record_to_model_string(record: Record) -> List[str]:
-        """
-        Utility method to parse the information from a slims record into
-        pydantic fields.
-        Parameters
-        ----------
-        record : Record
-          A record pulled from the slims db. Easiest way to pull a record is
-          using:
-          record = slims.fetch(
-                   "Content", is_not_null("cntn_id"), start=0, end=1
-          )[0]
-
-        Returns
-        -------
-        List[str]
-          A list of string representations of pydantic fields
 
-        """
-        columns_info = [
-            ContentsTableColumnInfo.model_validate(c)
-            for c in record.json_entity["columns"]
-        ]
-        return [c.map_to_field_str() for c in columns_info]
+class InstrumentTableRow(SlimsTableRow):
+    """A record pulled from slims Instrument Table."""
 
-    @classmethod
-    def from_record(cls, record: Record):
-        """Create a ContentsTableRow from a SLIMS Record"""
-        field_names = cls.model_fields
-        field_values = {}
-        for field_name in field_names:
-            record_value = getattr(record, field_name, None)
-            if record_value is not None:
-                record_value = record_value.value
-            field_values[field_name] = record_value
-        return cls(**field_values)
+    nstr_created_On: Optional[datetime] = Field(
+        None, title="Created on", description="timeZone: America/Los_Angeles"
+    )
+    nstr_fk_group: Optional[str] = Field(None, title="Group")
+    groupPkFieldName: Optional[str] = Field(None, title="Group Field Name")
+    nstr_fk_instrumentType: Optional[int] = Field(
+        None, title="Instrument Type"
+    )
+    nstr_fk_calibrationRun: Optional[bool] = Field(
+        None, title="Calibration Run"
+    )
+    icon: Optional[str] = Field(None, title="Icon")
+    stts_runstepSelection: Optional[bool] = Field(
+        None, title="Run Step Selection"
+    )
+    nstp_canBeCalibrated: Optional[bool] = Field(
+        None, title="Can be Calibrated"
+    )
+    rdrc_cf_fk_instrument_display: Optional[str] = Field(
+        None, title="Rdrc Instrument Display"
+    )
+    stts_name: Optional[str] = Field(None, title="Stts Name")
+    rslt_cf_fk_instrument_display: Optional[str] = Field(
+        None, title="Rslt Instrument Display"
+    )
+    rslt_cf_fk_balance_display: Optional[str] = Field(
+        None, title="Balance Display"
+    )
+    assignedGroupPk: Optional[str] = Field(None, title="Assigned Group")
+    rslt_cf_fk_injectionDevice_display: Optional[str] = Field(
+        None, title="Injection Device Display"
+    )
+    nstr_modifiedOn: Optional[datetime] = Field(
```

Comment:
We should consider making this a pydantic AwareDatetime type if the timezone is fixed to LA.